hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -389,6 +415,481 @@ __attribute__((section(".noxip"))) int _imxrt_setLevelLPCG(int clock, int level)
+		{ pctl_clk_lpuart8, mux_clkroot_lpuart8_osc24mout, 1, 1 },
+		{ pctl_clk_lpuart9, mux_clkroot_lpuart9_osc24mout, 1, 1 },
+		{ pctl_clk_lpuart10, mux_clkroot_lpuart10_osc24mout, 1, 1 },
+		{ pctl_clk_lpuart11, mux_clkroot_lpuart11_osc24mout, 1, 1 },
+		{ pctl_clk_lpuart12, mux_clkroot_lpuart12_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c1, mux_clkroot_lpi2c1_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c2, mux_clkroot_lpi2c2_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c3, mux_clkroot_lpi2c3_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c4, mux_clkroot_lpi2c4_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c5, mux_clkroot_lpi2c5_osc24mout, 1, 1 },
+		{ pctl_clk_lpi2c6, mux_clkroot_lpi2c6_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi1, mux_clkroot_lpspi1_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi2, mux_clkroot_lpspi2_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi3, mux_clkroot_lpspi3_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi4, mux_clkroot_lpspi4_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi5, mux_clkroot_lpspi5_osc24mout, 1, 1 },
+		{ pctl_clk_lpspi6, mux_clkroot_lpspi6_osc24mout, 1, 1 },
+		{ pctl_clk_emv1, mux_clkroot_emv1_osc24mout, 1, 1 },
+		{ pctl_clk_emv2, mux_clkroot_emv2_osc24mout, 1, 1 },
+		{ pctl_clk_enet1, mux_clkroot_enet1_osc24mout, 1, 1 },
+		{ pctl_clk_enet2, mux_clkroot_enet2_osc24mout, 1, 1 },
+		{ pctl_clk_enet_qos, mux_clkroot_enet_qos_osc24mout, 1, 1 },
+		{ pctl_clk_enet_25m, mux_clkroot_enet_25m_osc24mout, 1, 1 },
+		{ pctl_clk_enet_timer1, mux_clkroot_enet_timer1_osc24mout, 1, 1 },
+		{ pctl_clk_enet_timer2, mux_clkroot_enet_timer2_osc24mout, 1, 1 },
+		{ pctl_clk_enet_timer3, mux_clkroot_enet_timer3_osc24mout, 1, 1 },
+		{ pctl_clk_usdhc1, mux_clkroot_usdhc1_osc24mout, 1, 1 },
+		{ pctl_clk_usdhc2, mux_clkroot_usdhc2_osc24mout, 1, 1 },
+		{ pctl_clk_asrc, mux_clkroot_asrc_osc24mout, 1, 1 },
+		{ pctl_clk_mqs, mux_clkroot_mqs_osc24mout, 1, 1 },
+		{ pctl_clk_mic, mux_clkroot_mic_osc24mout, 1, 1 },
+		{ pctl_clk_spdif, mux_clkroot_spdif_osc24mout, 1, 1 },
+		{ pctl_clk_sai1, mux_clkroot_sai1_osc24mout, 1, 1 },
+		{ pctl_clk_sai2, mux_clkroot_sai2_osc24mout, 1, 1 },
+		{ pctl_clk_sai3, mux_clkroot_sai3_osc24mout, 1, 1 },
+		{ pctl_clk_sai4, mux_clkroot_sai4_osc24mout, 1, 1 },
+		/* NOTE: "pctl_clk_gpu2d" not using video peripheral - clock turned off */
+		{ pctl_clk_gpu2d, mux_clkroot_gpu2d_videopllout, 2, 0 },
+		{ pctl_clk_lcdif, mux_clkroot_lcdif_osc24mout, 1, 1 },
+		{ pctl_clk_lcdifv2, mux_clkroot_lcdifv2_osc24mout, 1, 1 },
+		{ pctl_clk_mipi_ref, mux_clkroot_mipi_ref_osc24mout, 1, 1 },
+		{ pctl_clk_mipi_esc, mux_clkroot_mipi_esc_osc24mout, 1, 1 },
+		{ pctl_clk_csi2, mux_clkroot_csi2_osc24mout, 1, 1 },
+		{ pctl_clk_csi2_esc, mux_clkroot_csi2_esc_osc24mout, 1, 1 },
+		{ pctl_clk_csi2_ui, mux_clkroot_csi2_ui_osc24mout, 1, 1 },
+		{ pctl_clk_csi, mux_clkroot_csi_osc24mout, 1, 1 },
+		{ pctl_clk_clko1, mux_clkroot_cko1_osc24mout, 1, 1 },
+		{ pctl_clk_clko2, mux_clkroot_cko2_osc24mout, 1, 1 },
+	};
+
+	for (unsigned n = 0; n < sizeof(clktree) / sizeof(clktree[0]); n++) {
```

Comment:
We keep C89 way of life in kernel and plo ;)
```suggestion
	for (n = 0; n < sizeof(clktree) / sizeof(clktree[0]); n++) {
```

Reply:
Done.